app/ui/client/views/app/room.js

```diff
@@ -970,19 +970,19 @@ Meteor.startup(() => {
 		// salva a data da renderização para exibir alertas de novas mensagens
 		$.data(this.firstNode, 'renderedAt', new Date());
 
-		const webrtc = WebRTC.getInstanceByRoomId(template.data._id);
-		if (webrtc) {
-			this.autorun(() => {
-				const remoteItems = webrtc.remoteItems.get();
-				if (remoteItems && remoteItems.length > 0) {
-					this.tabBar.open('members-list');
-				}
-
-				if (webrtc.localUrl.get()) {
-					this.tabBar.open('members-list');
-				}
-			});
-		}
+		// const webrtc = WebRTC.getInstanceByRoomId(template.data._id);
```

Comment:
@ggazzo Do you thinking we should open the webRTC view in a different tab, instead of open in the User Info tab?